samples/es/invoice-es-es-freelance.json

```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "https://gobl.org/draft-0/envelope",
 	"head": {
-		"uuid": "8cb8f9e3-e8c7-11ec-96bb-e2c0624703d9",
+		"uuid": "c68abf80-eb63-11ec-9fd9-e2c0624703d9",
```

Comment:
why do we need all this changes of UUID and signatures?

Reply:
They're updated because of the tests... originally these JSON were ignored. Not sure what to do about it yet as it's useful to have examples in the code.

Reply:
okay, figured out a way to do that now and avoid updates.